lib/p2p/errors.ts

```diff
@@ -29,6 +32,10 @@ const errors = {
     message: `could not retrieve external IP: ${err.message}`,
     code: errorCodes.EXTERNAL_IP_UNRETRIEVABLE,
   }),
+  CONNECTING_RETRIES_MAX_PERIOD_EXCEEDED: {
+    message: `Connection retries max period exceeded`,
```

Comment:
Maybe make this a bit more descriptive, like "Connection retry attempts to peer exceeded maximum time allotment"